packages/expo-blur/src/BlurView.tsx

```diff
@@ -1,41 +1,24 @@
-import { NativeModulesProxy, requireNativeViewManager } from 'expo-modules-core';
+import { requireNativeViewManager } from 'expo-modules-core';
 import * as React from 'react';
-import { findNodeHandle, View, StyleSheet } from 'react-native';
+import { View, StyleSheet } from 'react-native';
 
-import { BlurProps, BlurTint, ComponentOrHandle } from './BlurView.types';
+import { BlurProps } from './BlurView.types';
 
 export default class BlurView extends React.Component<BlurProps> {
```

Comment:
Since most of the class related code has been removed, can we refactor this component to the functional one?

Reply:
See the current implementation. I've researched the topic and there's no other way to achieve what's I was trying to do (ability to pass `ref={useRef}` prop and properly separate view-based props and blur-based props) 🤷